test/visibility/private/variables/accessSiblingPrint.chpl

```diff
@@ -6,7 +6,7 @@ module M2 {
   use M1;
   proc main() {
     // ignoreInternalModules=true, ignoreBuiltInModules=true
-    __primitive(c"get visible symbols", true, true);
+    __primitive("get visible symbols", true, true);
```

Comment:
I'm also surprised that this test and the other test below it don't have a corresponding .good file update